vhdbuilder/packer/test/run-test.sh

```diff
@@ -11,7 +11,7 @@ TEST_VM_ADMIN_PASSWORD="TestVM@$(date +%s)"
 set -x
 
 if [ "$OS_TYPE" == "Linux" ]; then
-  if [ "$IMG_SKU" == "20_04-lts-cvm" ] || [ "$OS_VERSION" == "V1" ] && [ "$OS_SKU" == "CBLMariner" ]; then
+  if [ "$IMG_SKU" == "20_04-lts-cvm" ] || [ "$OS_VERSION" == "V1" ]; then
```

Comment:
Nit: Not building Mariner V1 any more, we don't need to check `|| [ "$OS_VERSION" == "V1" ]`

Reply:
Fixed.